src/briefcase/commands/base.py

```diff
@@ -149,6 +151,12 @@ class BaseCommand(ABC):
     # introduced for a platform such that older versions of a template are incompatible
     platform_target_version: str | None = None
 
+    # We specify the default app template as a class attribute so that it can be
+    # overwritten by properties of subclasses. This is necessary since if we had it as
+    # an instance attribute, it would cause an AttributeError since the property would
+    # not have a setter.
+    app_template_url = None
```

Comment:
I can see that `create` is removing the template details from the context, but I don't think there's a particularly *strong* reason why that is the case. It's more an accident of history - we originally did a little bit of cleanup of the template context; we reverted that for `new` so that we could output the specific template version in the template, but we never made the corresponding change to `create` (and, for some reason, we used "template_source", rather than "template" for the template variable). It can't think of any reason it would hurt to pass in extra template variables - if anything, it would be beneficial, as we could include the template version in the platform templates - so for the sake of simplifying `generate_template`, we might as well include them.

As for `fallback_template` - you're correct there's no existing usage of `a=x or y`; that's because the idiom we use is `a = x if x else y`.